AnkiDroid/src/main/java/com/ichi2/anki/export/ActivityExportingDelegate.java

```diff
@@ -145,7 +145,7 @@ public void emailFile(String path) {
                 .setType("application/apkg")
                 .setStream(uri)
                 .setSubject(mActivity.getString(R.string.export_email_subject, attachment.getName()))
-                .setHtmlText(mActivity.getString(R.string.export_email_text))
+                .setHtmlText(mActivity.getString(R.string.export_email_text_new, mActivity.getString(R.string.link_manual), mActivity.getString(R.string.link_distributions)))
```

Comment:
I can't see how this is correct :thinking: 
I think you need to `setHtmlText` with the result of a `String.format` *on* a `getString` of the translated string, substituting in the `getString`s of the links?

I believe like this, though this is just coded into a web text box, may not compile + needs testing

```suggestion
                .setHtmlText(String.format(mActivity.getString(R.string.export_email_text_new), mActivity.getString(R.string.link_manual), mActivity.getString(R.string.link_distributions))
```

Reply:
https://developer.android.com/reference/android/content/Context#getString(int,%20java.lang.Object...)

> Returns a localized formatted string from the application's package's default string table, substituting the format arguments as defined in Formatter and String.format(String, Object...).

I tested this locally (which was not trivial, since at first I did not find the sharing this was used :/), and the string is filled as expected with my original code.

Reply:
Fascinating, I believe you (both that testing it was probably a pain, and that it works) but that means I have some incorrect understanding in my head. I'll re-read the diff here and think about it again, thanks for reporting your results

Reply:
This looks correct to me from casual observation: the format strings should be inside `getString`, as they're placeholders in the .xml resource

Reply:
I understand from David that getString is better, and after carefully re-reading, I see I was mis-counting open/closed parens leading to my further confusion as I thought they were balanced differently then they were. This is great